webonise.txt

```diff
@@ -50,3 +50,5 @@ int main() {
   return 0;
 }
 
+
+I am adding staging changes here --"staging changes"
```

Comment:
modify staging changes to "something else" in same commit. do not add separate commit

Reply:
Do need to change the commit message or the words inside the file..?